package-lock.json

```diff
@@ -31,31 +31,31 @@
             }
         },
         "@microsoft/sarif-multitool": {
-            "version": "2.2.2",
-            "resolved": "https://registry.npmjs.org/@microsoft/sarif-multitool/-/sarif-multitool-2.2.2.tgz",
-            "integrity": "sha512-mid4xKQv5QkrbZtH+V0Tlwl3aCneZtAEoggXqY7klwuza1Zn1uI7CkRqq8aDiv+fWR/a6acAbagEogwzfRVY8A==",
+            "version": "2.2.3",
```

Comment:
This once again is because the package-lock json file in master has not been updated yet.